DisplayOptionsPanel.c

```diff
@@ -22,11 +22,11 @@ in the source distribution for its full text.
 #include "ScreensPanel.h"
 
 
-static const char* const DisplayOptionsFunctions[] = {"      ", "      ", "      ", "      ", "      ", "      ", "      ", "      ", "      ", "Done  ", NULL};
+static const char* const DisplayOptionsFunctions[] =       {"      ", "      ", "      ", "      ", "      ", "      ", "      ", "      ", "      ", "Done  ", NULL};
 
-static const char* const DisplayOptionsDecIncFunctions[] = {"Dec   ", "Inc   ", "                                                      ", "Done  ", NULL};
-static const char* const DisplayOptionsDecIncKeys[] = {"- ", "+ ", "  ", "F10", NULL};
-static const int DisplayOptionsDecIncEvents[] = {'-', '+', ERR, KEY_F(10)};
+static const char* const DisplayOptionsDecIncFunctions[] = {"      ", "      ", "      ", "      ", "      ", "      ", " Dec  ", " Inc  ", "      ", "Done  ", NULL};
```

Comment:
Items are left-aligned within the button label …
```suggestion
static const char* const DisplayOptionsDecIncFunctions[] = {"      ", "      ", "      ", "      ", "      ", "      ", "Dec   ", "Inc   ", "      ", "Done  ", NULL};
```

Reply:
I like the middle alignment better when we have the space...

Reply:
It's just inconsistent with everywhere else in the UI though …

Reply:
Ugly wins. You may only wear grey cat ears from now on!